src/rpsapp.js

```diff
@@ -0,0 +1,93 @@
+import getThrow from './rpstest.js';
+
+const throwButton = document.getElementById('throwbutton');
+
+ 
+const winCount = document.getElementById('win-count');
+const lossCount = document.getElementById('loss-count');
+const tiesCount = document.getElementById('ties-count');
+const computerImage = document.getElementById('computer-choice');
+const addMessage = document.getElementById('message');
+
+let moneyRemaining = 50;
+let updateMoney = document.getElementById('moneyremaining');
+let amount = document.getElementById('amount');
+let currentBet = 0;
+let bet = document.getElementById('bet');
+updateMoney.textContent = `YOU HAVE $${moneyRemaining} TO BET`;
+
+let wins = 0;
+let losses = 0;
+let ties = 0;
+
+let message = '';
+// throw rock, paper, or scissors, based off random number. Function onclick of a random number.
```

Comment:
In the future, please delete any comments and commented out code before submitting